lib/jekyll/commands/new.rb

```diff
@@ -43,7 +43,10 @@ def process(args, options = {})
 
         def create_blank_site(path)
           Dir.chdir(path) do
-            FileUtils.mkdir(%w(_layouts _posts _drafts))
+            FileUtils.mkdir(%w(_data _drafts _includes _layouts _posts _sass assets assets/css))
+            FileUtils.touch("_config.yml")
+            FileUtils.touch("_sass/main.scss")
+            FileUtils.touch("assets/css/main.scss")
```

Comment:
You shouldn't just `touch` these files.. They're useless that way..
* `_config.yml` should never be empty. It will not validate otherwise.
* `assets/css/main.scss` should `import "main"` otherwise there's no point having a `_sass/main.scss`.
* Might as well *add something* to `_sass/main.scss` since its just sitting there..

Reply:
empty `config.yml` is fine, I tested the generated *blank* scaffold, and no error on `jekyll build`.

Reply:
Strange..
https://github.com/jekyll/jekyll/blob/153f24c4e4e00574fce074d8cc261a5f754a0e58/lib/jekyll/configuration.rb#L304-L309
https://github.com/jekyll/jekyll/blob/153f24c4e4e00574fce074d8cc261a5f754a0e58/lib/jekyll/configuration.rb#L171-L173

Reply:
I believe YAML works in mysterious ways here where an empty file counts as a `Hash`.

Reply:
okay...
So empty config files default to an empty Hash..
https://github.com/jekyll/jekyll/blob/153f24c4e4e00574fce074d8cc261a5f754a0e58/lib/jekyll/configuration.rb#L134-L135

Reply:
@ashmaroli addressed your concerns, `--blank` now provides a default layout linking to a generated CSS, and  *blank* config file. I don't think we should go further.